parser/src/cfg/mod.rs

```diff
@@ -798,6 +802,11 @@ pub fn parse_cfg_raw_string(
         .collect::<Vec<_>>();
     parse_aliases(&alias_exprs, s, &env_vars)?;
 
+    let start_action = cfg
+        .start_alias
+        .as_ref()
+        .and_then(|start| s.aliases.get(start).copied());
```

Comment:
An error should be triggered if `start_alias` is defined but the `get` call finds nothing.

Reply:
Yes, for sure.  pushed.